plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerRegistryChecker.java

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2015 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.plugin.docker.client;
+
+import com.google.inject.name.Named;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/**
+ * Checks that docker registry is available.
+ *
+ * @author Yevhenii Voevodin
+ */
+@Singleton
+public class DockerRegistryChecker {
+
+    private static final Logger LOG = LoggerFactory.getLogger(DockerRegistryChecker.class);
+
+    @Inject
+    private DockerConnector connector;
+
+    @Inject
+    @Named("docker.registry.auth.url")
+    private String registry;
+
+    /**
+     * Checks that registry is available and if it is not - logs warning message.
+     */
+    @PostConstruct
+    private void checkRegistryIsAvailable() throws IOException {
+        if (!probeUrl("http://" + registry) && !probeUrl("https://" + registry)) {
```

Comment:
If you mean `docker.registry.auth.url` property then my answer is - no we can't, because 
docker will not be able to make push to the registry tagged as http://localhost:5000. In this case we can try to use several different properties.